libs/coin-modules/coin-tron/src/logic/craftTransaction.ts

```diff
@@ -1,22 +1,31 @@
 import { TransactionIntent } from "@ledgerhq/coin-framework/api/index";
-import { decode58Check } from "../network/format";
 import BigNumber from "bignumber.js";
 import { craftStandardTransaction, craftTrc20Transaction } from "../network";
+import { decode58Check } from "../network/format";
 import { TronToken } from "../types";
+import { feesToNumber } from "./utils";
 
 export async function craftTransaction(
```

Comment:
For tron, we will also need:
 - `memo` (`raw_data.data`)
 - `expiration`
(can be done in a later PR)